src/server/index.ts

```diff
@@ -14,13 +14,18 @@ interface Body {
 
 const build = () => {
   const server = fastify({ logger: logger })
-  server.setErrorHandler(function (error: any, request: FastifyRequest, reply: FastifyReply) {
+
+  server.setErrorHandler(function(error: any, request: FastifyRequest, reply: FastifyReply) {
     this.log.error(error)
 
     if (error?.name === 'SyntaxError') {
       reply.code(400)
     }
 
+    if (error?.name === 'SqliteError') {
+      reply.code(400)
+    }
```

Comment:
Should we able to send back also the message in the body so we can pass it to the UI? 
example error message could be nice to have in the UI:

```json
{"statusCode":500,"code":"SQLITE_ERROR","error":"Internal Server Error","message":"no such table: shifts"}
```

FYI, right now we handle it this way:
```ruby
## components/reports/app/services/reports/tentacle_connector.rb
raise InvalidQuery, "query: #{query} || body: #{res.body}" if res.code == 400

## components/reports/app/controllers/reports/reports_controller.rb
    rescue Reports::TentacleConnector::InvalidQuery => e
      render json: { error: e.message }, status: :unprocessable_entity
## 
```

Reply:
Won't this be handled by the final `reply.send(error)` Omar?